flask_of_oil/jwt_validator.py

```diff
@@ -81,6 +83,21 @@ def validate(self, jwt):
         # noinspection PyBroadException
         try:
             jws.verify_compact(jwt, self.jwks)
+        except NoSuitableSigningKeys as e:
+            now = datetime.now().timestamp()
+            if self.lastSuccessfulJWKSFetch + self.jwksFetchCooldown < now:
```

Comment:
`None + 30` will cause an exception, no?

Reply:
Good catch. I was thinking in PHP/JS terms...